app/views/dashboard/work_versions/_show_fields.html.erb

```diff
@@ -0,0 +1,74 @@
+<p>
+  <strong><%= WorkVersion.human_attribute_name('version_name') %>:</strong>
+  <%= work_version.version_name %>
+</p>
```

Comment:
@rschenk extra credit if you want to reformat these in a more bootstrap-ish way, such as a definition list. Up to you though...

Reply:
OnE gIaNt ViM mAcRo?!?!?!?